ui/src/main/webapp/wizards/sources/stages.js

```diff
@@ -162,7 +163,7 @@ const ConfirmationStageView = ({ messages, selectedSource, persistConfig, source
     <Description>
         Please give your source a unique name, confirm details, and press finish to create source.
       </Description>
-    <ConstrainedInput id='sourceName' label='Source Name' />
+    <ConstrainedInput id='sourceName' label='Source Name' autoFocus />
```

Comment:
Material UI passes attributes down to the input element, which is what recognizes the `autofocus` attribute.